web_navbar_chameleon/view/assets.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2019 Alexandre Díaz
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<odoo>
+    <template id="assets_backend" name="Web Navbar Chameleon Assets"
+              inherit_id="web.assets_backend">
+        <xpath expr=".">
+            <link rel="stylesheet" type="text/scss"
+                  t-attf-href="/web_navbar_chameleon/static/src/scss/web_navbar_chameleon.#{request.env.user and request.env.user.company_id.id or '1'}.gen.scss"
```

Comment:
to prevent caching we should store a timestamp when styles are updated and use it in the URL